PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1816,6 +1816,35 @@ bool Notepad_plus::findInFilelist(std::vector<generic_string> & fileNames)
 
 	const bool isEntireDoc = true;
 
+	size_t MaxLineNum = 0;
+	for (size_t i = 0; i < filesCount; ++i)
+	{
+		if (progress.isCancelled()) break;
+
+		bool closeBuf = false;
+		BufferID id = MainFileManager.getBufferFromName(fileNames.at(i).c_str());
+		if (id == BUFFER_INVALID)
+		{
+			id = MainFileManager.loadFile(fileNames.at(i).c_str());

```

Comment:
Loading every file to search (only for the aligning the line numbers) twice will make the performance issue in Find in Files feature.